desktop/core/ext-py3/boto-2.49.0/boto/sdb/item.py

```diff
@@ -62,7 +62,7 @@ def startElement(self, name, attrs, connection):
     def decode_value(self, value):
         if self.encoding == 'base64':
             self.encoding = None
-            return base64.decodestring(value)
+            return base64.decodebytes(value)
```

Comment:
Not sure how can we test this one!

Reply:
decodebytes is alias of decodestring.

Reply:
Suggestion: 
Might be possible to test via enabling S3 using access key and secret in python3.9 env.
Use breakpoint or other methods to see if any S3 browser functionality makes it come here and it works fine.

Reply:
I tried most of the operations but not able to hit this point.